swagger.json

```diff
@@ -16189,6 +16189,50 @@
                 }
             }
         },
+        "Binary": {
+            "type": "object",
+            "properties": {
+                "content": {
+                    "format": "string",
+                    "type": "string"
+                },
+                "contentType": {
+                    "format": "string",
+                    "type": "string"
+                },
```

Comment:
this used to be an enum
https://github.com/Keyclic/app-sdk/pull/27/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-10cfbe600d3f92e886aea1d00df1367f9899549b1a3235442e5191b66ced7e71L18595-L18611

Reply:
We will try to get the enum again, with a next swagger definition.